web/components/contract/contract-tabs.tsx

```diff
@@ -81,8 +82,10 @@ export function ContractTabs(props: {
     <Tabs
       tabs={[
         { title: 'Comments', content: commentActivity },
-        { title: 'Bets', content: betActivity },
-        ...(!user || !userBets?.length
+        ...(outcomeType === 'NUMERIC'
+          ? []
+          : [{ title: 'Bets', content: betActivity }]),
+        ...(!user || !userBets?.length // || outcomeType === 'NUMERIC'
           ? []
```

Comment:
Do we want to eventually show others' bets in Bets tab?

Also, remove extraneous comment `// || outcomeType === 'NUMERIC'`

Reply:
I can re-enable the bests tab.